nixos/modules/virtualisation/sysbox.nix

```
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.virtualisation.sysbox;
+in
+
+{
+  ###### interface
+
+  options.virtualisation.sysbox = {
+    enable =
+      mkOption {
+        type = types.bool;
+        default = false;
+        description =
+          lib.mdDoc ''
+            This option enables sysbox
```

Would be nice to be a little bit more precise what this does =). 
e.g. 

```suggestion
            Enables the container runtime `sysbox-runc` from `nestybox`. 
            Starts a `sysbox-mgr` manager service and makes the runtime available 
            for `docker` to be used in `docker run --runtime=sysbox-runc ...`.
```